test/clj/rems/test_browser.clj

```diff
@@ -740,24 +746,29 @@
         (btu/wait-page-loaded)
         (is (btu/visible? {:tag :label :class :application-field-label :fn/has-text "Option list (EN)"}))))
 
+
+
     (testing "fetch form via api"
       (let [form-id (Integer/parseInt (last (str/split (btu/get-url) #"/")))]
         (is (= {:form/id form-id
                 :organization {:organization/id "nbn" :organization/name {:fi "NBN" :en "NBN" :sv "NBN"} :organization/short-name {:fi "NBN" :en "NBN" :sv "NBN"}}
                 :form/title "Form editor test"
                 :form/fields [{:field/placeholder {:fi "" :en "" :sv ""}
                                :field/title {:fi "Description (FI)" :en "Description (EN)" :sv "Description (SV)"}
+                               :field/info-text {:en "", :fi "", :sv ""}
```

Comment:
Does this actually mean that one will always get an info-text even if the administrator left it empty? I think it would be cleanest not to return one at all if it was not set (since it's optional).

Reply:
I am doing the last changes and it looks like tests fail without these fields